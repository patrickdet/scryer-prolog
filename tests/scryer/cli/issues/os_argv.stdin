os:argv(V).
os:argv(["test1"|_]).
os:argv(["--"|_]).
os:argv([V, "--"|_]).
os:argv(["test2"|_]).
os:argv([]).
