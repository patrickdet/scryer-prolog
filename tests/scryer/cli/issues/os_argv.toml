# https://github.com/mthom/scryer-prolog/pull/2263#issuecomment-1874400820
args = ["-f", "--no-add-history", "--", "test1", "--", "test2", "--"]
