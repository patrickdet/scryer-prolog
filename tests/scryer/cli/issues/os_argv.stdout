   V = ["test1","--","test2","--"].
   true.
   false.
   V = "test1".
   false.
   false.
